src/python/pants/backend/python/subsystems/python_native_code.py

```diff
@@ -47,6 +48,7 @@ def subsystem_dependencies(cls):
     return super(PythonNativeCode, cls).subsystem_dependencies() + (
       NativeToolchain.scoped(cls),
       PythonSetup.scoped(cls),
```

Comment:
This highlight what looks like an already broken scoped here vs global in dump requirements. Not your issue but @cosmicexplorer and @CMLivingston may want to sort why scoped and why not consistent use.

Reply:
Hm. I was wondering about that too.